HW_1/Task_1.py

```diff
@@ -0,0 +1,327 @@
+'''1. За день машина проезжает n километров.  
+Сколько дней нужно, чтобы проехать маршрут длиной m километров?  
+При решении этой задачи нельзя пользоваться условной инструкцией if и циклами.
+
+**Input:**
+
+n = 700
+
+m = 750
+
+**Output:**
+
+2'''
+
+
+def distance():
+    print('Задание 1')
+    n = int(input())
+    m = int(input())
+    print((m-1)//n+1)
+
+# distance()
+
+
+'''2. В некоторой школе решили набрать три новых математических класса и оборудовать кабинеты для них новыми партами.  
+За каждой партой может сидеть два учащихся. Известно количество учащихся в каждом из трех классов.  
+Выведите наименьшее число парт, которое нужно приобрести для них.
+
+**Input:**
+
+20
+
+21
+
+22
+
+**Output:**
+
+32'''
+
+
+def kolichestvo_part(a):
+    return (a-1)//2+1
+
+
+def school():
+    print('Задание 2')
+    a = int(input())
+    b = int(input())
+    c = int(input())
+    print(kolichestvo_part(a)+kolichestvo_part(b)+kolichestvo_part(c))
+
+# school()
+
+
+'''3. Требуется вывести все целые степени двойки (т.е. числа вида 2k), не превосходящие числа N.  
+В решении дллжно быть не более 5 строк кода'''
+
+
+def dvoiki():
+    print('Задание 3')
+    N = float(input())
+    k = 0
+    while (2**k) < N:
+        print(k)
+        k += 1
+
+# dvoiki()
+
+
+'''4. Дана последовательность из N целых чисел и число K.  
+Необходимо сдвинуть всю последовательность (сдвиг - циклический) на K элементов вправо, K – положительное число.  
+Не использовать циклы.
+
+Input:   [1, 2, 3, 4, 5] k = 3
+
+Output:  [4, 5, 1, 2, 3]'''
+
+
+def list_4():
+    print('Задание 4')
+    list_1 = list(map(int, input().split(',')))
```

Comment:
Не  рекомендуется использовать функцию list для преобразования коллекций. Считается,  что на работает медленно.
```suggestion
    list_1 = [*map(int, input().split(','))]
```